osu.Game.Rulesets.Mania.Tests/ManiaDifficultyCalculatorTest.cs

```diff
@@ -13,7 +13,7 @@ public class ManiaDifficultyCalculatorTest : DifficultyCalculatorTest
     {
         protected override string ResourceAssembly => "osu.Game.Rulesets.Mania";
 
-        [TestCase(2.3683365342338796d, "diffcalc-test")]
+        [TestCase(2.9183460420403056d, "diffcalc-test")]
```

Comment:
Revert, if this changes in a non-mania-related change then your code is broken.

Reply:
Done